.github/workflows/examples-tizen.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Build example - Tizen
+
+on:
+    push:
+    pull_request:
+
+jobs:
+    tizen:
+        name: Tizen
+        env:
+            BUILD_TYPE: tizen
+
+        runs-on: ubuntu-latest
+        if: github.actor != 'restyled-io[bot]'
+
+        container:
+            image: connectedhomeip/chip-build-tizen:latest
+            volumes:
+                - "/tmp/bloat_reports:/tmp/bloat_reports"
+                - "/tmp/output_binaries:/tmp/output_binaries"
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+              with:
+                  submodules: true
+            - name: Build example Tizen lighting app
+              run:
+                  scripts/examples/gn_tizen_example.sh examples/lighting-app/linux
```

Comment:
any chance you can add support for scripts/build/build-examples.py and then this command becomes:

```
                ./scripts/run_in_build_env.sh \
                  "./scripts/build/build_examples.py --platform tizen build"
```

Reply:
I added tizen.py to build the example by using build_examples.py.
Thank you.